<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RxProjectClass</class>
 <widget class="QMainWindow" name="RxProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>471</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>829</width>
    <height>471</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>829</width>
    <height>471</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Fixedsys</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>The RxProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>829</width>
     <height>471</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>829</width>
     <height>471</height>
    </size>
   </property>
   <widget class="QListWidget" name="listWidgetConnected">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnected">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connectes ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrentUser">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Connected to a client</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidgetCore">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>641</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabFileSystem">
     <attribute name="title">
      <string>FileSytem</string>
     </attribute>
     <widget class="QListWidget" name="listWidgetFileSystem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidgetFileSystem">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>371</width>
        <height>351</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabFileSystemNavig">
       <attribute name="title">
        <string>Navigation</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonCreateFile">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Create .txt File</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditCreateFile">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>351</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditCreateFile">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>179</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditRename">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonExec">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>191</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Exec</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonRename">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPaste">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Paste</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonCopy">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabUploadDownload">
       <attribute name="title">
        <string>Up/Down</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonDownload">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>351</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonUpload">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>351</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBarUpDown">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="labelUpDownInfos">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>230</y>
          <width>361</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nothing now ...</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabKeyLogger">
     <attribute name="title">
      <string>KeyLogger</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowserKeyLog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelKeyLogOnOff">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>15</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>KeyLogging OFF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStartKeyLog">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Key Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStopKeyLog">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Key Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClearKeyLog">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>140</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSaveKeyLog">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>310</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSaveKeyLog">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabNetMonitor">
     <attribute name="title">
      <string>Net Monitor</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowserMonitor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAbout">
     <attribute name="title">
      <string>About ...</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditVersion">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelVersion">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
